ArrtSource/Storage/UI/StorageBrowserModel.cpp

```diff
@@ -281,6 +281,17 @@ bool StorageBrowserModel::IsSrcAsset(const QString& file)
     return formats.indexOf(extension) >= 0;
 }
 
+bool StorageBrowserModel::IsPointCloud(const QString& file)
```

Comment:
I would call this function "IsSingleFileAsset" because this code path does not only apply to Point Clouds but in theory to all file types that may only consist of a single file. Conversely, we may support pointcloud file types that consist of multiple files in the future